qms-tests/sanity/tests/model/documents/document-content-page.ts

```diff
@@ -404,6 +453,19 @@ export class DocumentContentPage extends DocumentCommonPage {
     await this.checkDocumentStatus(DocumentStatus.DRAFT)
   }
 
+  async checkTeamMembersReviewNotExists (): Promise<void> {
```

Comment:
Minor: the naming is confusing as it checks not only review but approver/co-author assignment as well.

Reply:
hmmm true, I will rename it